tockloader/src/main.rs

```diff
@@ -25,8 +31,15 @@ async fn run() -> Result<(), TockloaderError> {
     match matches.subcommand() {
         Some(("listen", sub_matches)) => {
             let mut interface = build_interface(sub_matches)?;
```

Comment:
I don't think we need to create an interface for this. Especially since the `tock-process-console` already deals with serial communications. We will likely also need to merge these somehow, but I would leave them separate for now

Reply:
was previously using build_interface as a pre-ratatui way of viewing all your ports to know what to chose for the input for ratatui since ill be needing this ill prob get rid of it when i complete the ratatui serial changes.

but it is indeed not needed just temporarily convenient for me.